_posts/2017-06-21-Devoxx-pl.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+authors: [bart_blommaerts, jeff_mesens]
+title: "Devoxx Poland 2017"
+image: /img/2017-devoxx-pl/devoxx-poland.jpg
+tags: []
+category: Conference
+comments: true
+---
+# Devoxx Poland 2017
+_(This post is still in progress and will be updated on a regular base)_
+
+Krakow in the ICE Krakow Congress Centre.
+
+![ICE Krakow Congress Centre](/img/2017-devoxx-pl/venue-1.jpg){: .image height="420px" }
+![ICE Krakow Congress Centre foyer](/img/2017-devoxx-pl/venue-2.jpg){: .image height="420px" }
+
+We started off day 1 with the keynote in the absolutely phenomenal main room:
```

Comment:
We started off day 1 with the keynote in the absolutely, phenomenal main room
(comma)